base62.py

```diff
@@ -40,6 +72,25 @@ def decode(b):
 
     return v
 
+
+def decodebytes(s):
+    """Decodes a bytestring of base62 data into a bytes object.
+
+    :param s: A bytestring to be decoded in base62
+    :rtype: bytearray
+    """
+    _check_bytes_type(s)
+
+    decoded = decode(s.decode(DEFAULT_ENCODING))
+    buf = []
```

Comment:
Rather than creating a list that is later converted to a `bytearray`, just declare the bytearray here. `bytearray` support `append` and `reverse`.